docs/getting-developed/how-to-access-api.md

```diff
@@ -9,8 +9,8 @@
 
 | 注意事项                                     | 参考文档                                          |
 | -------------------------------------------- | ------------------------------------------------- |
-| 推荐使用各自语言的 SDK，而不是调用原生的接口 | [SDK 快速开始页面](../docs/getting-started)       |
-| 阅读 OpenAPI 介绍中开通相应服务              | [OpenAPI 如何开通](../docs/#如何开通)             |
+| 推荐使用各自语言的SDK，而不是调用原生的接口 | [SDK 快速开始页面](../docs/getting-started)       |
```

Comment:
🚫 **[AutoCorrect Lint]** <[AutoCorrect](https://github.com/huacnlee/autocorrect)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
```suggestion
| 推荐使用各自语言的 SDK，而不是调用原生的接口 |��SDK，而不是调用原生的接口 | [SDK 快速开始页面](../docs/getting-started)       |
```
<details><summary>reviewdog suggestion error</summary>GitHub comment range and suggestion line range must be same. L12-L12 v.s. L13-L13</details>